api/v1alpha3/azurecluster_types.go

```diff
@@ -48,6 +51,10 @@ type AzureClusterSpec struct {
 	// ones added by default.
 	// +optional
 	AdditionalTags Tags `json:"additionalTags,omitempty"`
+
+	// IdentityName is a reference to a AzureIdentity to be used when reconciling this cluster
+	// +optional
+	IdentityName *string `json:"identityName,omitempty"`
```

Comment:
This API change doesn't match what was proposed to https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/docs/proposals/20200720-single-controller-multitenancy.md#cluster-api-provider-azure-v1alpha3-types 

If changes to the design had to be made, let's first update the proposal so it matches the implementation.

Reply:
@devigned you mentioned you update the proposal? or I can do it if you haven't already changed it

Reply:
Will do. Just been short on time.

Reply:
let's make sure we update the proposal before merging this PR 

/hold

Reply:
also wondering if we need a change to clusterctl move before merging this in

Reply:
is the identity owned by the cluster? if so it should be moved with it already

should be pretty easy to validate that assumption

Reply:
I don't think its owned by the cluster, need to double check that